KEYPROTECT.md

```diff
@@ -128,23 +128,33 @@ The response will be in the following format, we will store this into the file `
 
 ## Putting the wrapped key into cluster
 
-We can then create a secret with this wrapped key with the secret type `type=kp-key` for keyprotect key
+We can then create a secret with this wrapped key with the secret type `type=kp-key` for keyprotect key, using the `ciphertext` field and `keyVersion.id` field above:
```

Comment:
```suggestion
We can then create a secret with this wrapped key with the secret type `type=kp-key` for keyprotect key, using the `ciphertext` field and `keyid` field above:
```

Reply:
seems like the code is now looking for `keyid`.
Assume this is root key, should we call it that? e.g. rootkeyid:

Reply:
yeah, that may be a bit more clean for the users

```
ibmcloud kp key wrap --help
NAME:
   wrap - Wrap a data encryption key.

USAGE:
  ibmcloud kp key wrap ROOT_KEY_ID [-p PLAINTEXT] [-a AAD_STRINGS] [-o FORMAT]
```

Reply:
so do we want to all caps `ROOT_KEY_ID` or `rootkeyid` or `root-key-id`. i am good with all, which is prefered

Reply:
we have the field 'ciphertext' so consistency would see it be 'rootkeyid' there 
i think the command line example are placeholders not litternal so ok to be different